intercom_flutter_web/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 xChange OÜ
```

Comment:
@v3rm0n is this license still valid?

Reply:
I'm not sure how it works. Initially the project was open sourced under this company and I forked it, but the copyright probably is still theirs so I kept it. It should not actually apply to any new code though AFAIK...

Reply:
I have used the same license for the intercom_flutter_platform_interface also.

Reply:
I guess this can be handled in the future if we need to change that, but I think new code should not have that license

Reply:
Which license do you want to put for intercom_flutter_web?